pkg/deploy/api/types.go

```diff
@@ -206,3 +206,16 @@ type DeploymentConfigList struct {
 	kapi.ListMeta `json:"metadata,omitempty" yaml:"metadata,omitempty"`
 	Items         []DeploymentConfig `json:"items" yaml:"items"`
 }
+
+type DeploymentConfigRollback struct {
+	kapi.TypeMeta `json:",inline" yaml:",inline"`
+	DeploymentRef *kapi.ObjectReference        `json:"deploymentRef,omitempty" yaml:"deploymentRef,omitempty"`
+	Spec          DeploymentConfigRollbackSpec `json:"spec,omitempty" yaml:"spec,omitempty"`
+}
+
+type DeploymentConfigRollbackSpec struct {
+	IncludeTriggers        bool `json:"includeTriggers,omitempty" yaml:"includeTriggers,omitempty"`
```

Comment:
You shouldn't add yaml anymore.  I would make these *not* omitempty since they convey to the user useful info.